### ISSUE TITLE
Rules for how the math is spoken

### ISSUE THREAD

**Issue by benetechMatt** [benetech/MathShare#999]

**Comment by alexrcabral** [comment]
@benetechMatt If you mean clarifications on how to speak math aloud depending on context (eg handling fractions and parentheses appropriately), this would be good to work with Neil on, since he did a lot of work on spoken math (eg ClearSpeak)

**Comment by jscholes** [comment]
Agreed with @alexrcabral here.  @NSoiffer and @sinabahram are best placed to provide input on this issue.  MathShare already utilises speech rules for screen reader users, so if appropriate they could be reused in some fashion.
